components/tikv_util/src/timer.rs

```diff
@@ -114,6 +115,10 @@ fn start_global_timer() -> Handle {
     rx.recv().unwrap()
 }
 
+fn start_global_timer() -> Handle {
```

Comment:
Instead of creating a new function, I suggest to change the place that use `start_global_timer`.

Reply:
fixed